### PR TITLE
Feature/scenario1 test2

### LINKED ISSUE #48: Add second test to scenario1
Following should be done:

1. Create infrastructure
1. Run balancing test
1. Drop one node
1. Check that all requests are still served
1. Restore the node
1. Run balancing test

Maybe requires fix for #42

### PR DESCRIPTION
**Pull request by outcatcher**
Resolve #48

### CODE REVIEW

File: scenarios/scenario1/test/single.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    status_code=$( curl -X -o /dev/null -q POST https://csm.outcatcher.com/telegraf -d "${infl_row}" -w "%{http_code}" )
+    if [[ status_code != 204 ]]; then
+        echo "Can't report status to telegraf"
+        exit 3
+    fi
+}
+
+archive=lb_test.tgz
+file_name=load_balancer_test
+start_test="./${file_name} ${bastion_eip}"
```
Comment (anton-sidelnikov) [inline scenarios/scenario1/test/single.sh]: bastion_eip not lb_eip?

File: scenarios/scenario1/test/single.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+scenario_dir=$(cd "${local_dir}/.."; pwd)
+echo "Scenario directory: ${scenario_dir}"
+cd "${scenario_dir}" || exit 1
+terraform init || exit $?
+
+ws_name="single"
+
+terraform workspace select ${ws_name} || terraform workspace new ${ws_name} || exit $?
+
+bastion_eip="80.158.3.174"
```
Comment (anton-sidelnikov) [inline scenarios/scenario1/test/single.sh]: this obsolete now

**Comment by anton-sidelnikov** [comment]
in playbooks/scenario1_stop_server_on_random_node.yml please change:
hosts: "{{ groups.scn1_gatewayed | random }}" to hosts: "{{ groups.gatewayed | random }}"

File: scenarios/scenario1/test/single.sh (hunk 2 above)
Comment (outcatcher) [reply scenarios/scenario1/test/single.sh]: this is required to retrieve public IPs in `post_build.sh`

File: scenarios/scenario1/test/single.sh [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+if [[ ! -e ${archive} ]]; then
+    wget -q -O ${archive} https://github.com/opentelekomcloud-infra/csm-test-utils/releases/download/v${version}/lb_test-${version}-linux.tar.gz
+    tar xf ${archive}
+fi
+
+prepare
+echo Preparation Finished
+lb_host="ecs-80-158-23-240.reverse.open-telekom-cloud.com"
+echo LB at ${lb_host}
+start_test="./load_balancer_test ${lb_host} 300"
```
Comment (anton-sidelnikov) [inline scenarios/scenario1/test/single.sh]: rebalance test?

**Pull request merged**